lib/business_time/business_days.rb

```diff
@@ -36,7 +36,7 @@ def before(time = Time.current)
     private
 
     def positive_days?
-      @days > 0
+      @days >= 0
```

Comment:
Please don’t. I would like to know what actually was the reason for this change, but in any case, the method named `positive_days?` **cannot be implemented like this**.

Introduce `non_negative_days?` or like.

Reply:
True! Applying.

actually It was just a starting point to find a solution for the kantox-flow errors in Rails 5.2, that is using the last version of this gem:

[here](https://github.com/kantox/business_time/blob/350ab66e865a1926501efc906e7eed36e329b72c/lib/business_time/business_days.rb#L23) instead of execute the `calculate_after` that is what it does in v0.8.1, it executes the `calculate_before`